.gitignore

```diff
@@ -8,3 +8,5 @@
 **__pycache__/
 *.pyc
 **db.sqlite3
+.coverage
+coverage.xml
```

Comment:
Consider removing the space before the `coverage.xml` entry to maintain consistency with the other entries.